### ISSUE TITLE
stability improvement: unittest for basic download

### ISSUE THREAD

**Issue by dev-rke** [pyload/pyload#1849]
Hi,

due to the update manager nearly everybody who runs this application will have problems if a plugin breaks basic functionality.
To improve stability in future, it should be useful to implement an unittest which tries to download small files (around 1-5MB) from a few different hosters in parallel.
Before pushing new commits the developers could check very fast if the basic implementation of downloading files still works and if not they directly see what breaks.

This should avoid breaking basic functionality and the test should run in less than a minute.
Also it should be simple to implement.
What do you think about?

**Comment by vuolter** [comment]
Agree. Plus, in 0.4.10 you'll be able to choose between stable updates or dev ones.
I'm really embarrassed by all that is happening...

**Comment by runningbits** [comment]
This seems to be more like an "integration test" or "functional test" (instead of a unit test) but automated tests are a good idea anyway...
